pkg/tfvars/openstack/ports.go

```diff
@@ -21,85 +9,3 @@ type terraformPort struct {
 	NetworkID string             `json:"network_id"`
 	FixedIP   []terraformFixedIP `json:"fixed_ips"`
 }
-
-func portTargetToTerraformPort(networkClient *gophercloud.ServiceClient, portTarget types_openstack.PortTarget) (terraformPort, error) {
```

Comment:
This is being used for the dual-stack support:
https://github.com/openshift/installer/pull/6797/files#diff-069c83ecc685bc97ef280d65021909c03ea67f51f67a998bb3a5cb647b8cf16bR183